client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -55,6 +59,12 @@ class CampaignAdminList extends Component {
         .then(() => {
           this.setBreadcrumbs();
           this.setState({ loading: false });
+        })
+        // Catch error and set Error message
+        .catch((e) => {
```

Comment:
Won't `e` have an actual HTTP status code attach to it?

Reply:
DONE